lib/std/crypto/25519/edwards25519.zig

```diff
@@ -235,18 +241,18 @@ pub const Edwards25519 = struct {
     /// Multiply an Edwards25519 point by a scalar without clamping it.
     /// Return error.WeakPublicKey if the resulting point is
```

Comment:
It's kind of weird to document only one of the 2 errors.
Id expect neither or both.

Reply:
Thanks, the comment has been updated.